### PR TITLE
chore(release): publish v2.1.0

### PR DESCRIPTION
**Pull request by petermetz**
Signed-off-by: Peter Somogyvari <peter.somogyvari@accenture.com>

**Pull Request Requirements**
- [x] Rebased onto `upstream/main` branch and squashed into single commit to help maintainers review it more efficient and to avoid spaghetti git commit graphs that obfuscate which commit did exactly what change, when and, why.
- [x] Have git sign off at the end of commit message to avoid being marked red. You can add `-s` flag when using `git commit` command. You may refer to this [link](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for more information.
- [x] Follow the Commit Linting specification. You may refer to this [link](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for more information. 

**Character Limit**
- [x] Pull Request Title and Commit Subject must not exceed 72 characters (including spaces and special characters).
- [x] Commit Message per line must not exceed 80 characters (including spaces and special characters).

**A Must Read for Beginners**
For rebasing and squashing, here's a [must read guide](https://github.com/servo/servo/wiki/Beginner's-guide-to-rebasing-and-squashing) for beginners.

### CODE REVIEW

**Comment by petermetz** [comment]
@hyperledger-cacti/cacti-maintainers bump

**Comment by sandeepnRES** [comment]
@petermetz Hey, is something wrong with mergefreeze? On the site it shows this PR is unfrozen, but mergrefreeze status isn't passed.

File: weaver/core/network/fabric-interop-cc/contracts/interop/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.0.0
\ No newline at end of file
+2.1.0
```
Comment (sandeepnRES) [inline weaver/core/network/fabric-interop-cc/contracts/interop/VERSION:1]: Hi @petermetz Can you run `go-gen-checksum` script? If already done please try again, check if there was some error? The go.mod and go.sum isn't updated anywhere

**Review by jagpreetsinghsasan — APPROVED** [review]
LGTM

**Review by outSH — APPROVED** [review]

File: weaver/core/network/fabric-interop-cc/contracts/interop/VERSION (hunk 1 above)
Comment (petermetz) [reply weaver/core/network/fabric-interop-cc/contracts/interop/VERSION:1]: @sandeepnRES I re-ran the script but it doesn't produce any diff unfortunately. Please see the output:

```sh
$ ./tools/go-gen-checksum.sh 2.1.0 .
Installing go-checksum...
Installed.
REPO: github.com/hyperledger/cacti
VERSION: 2.1.0
############# START weaver/core/network/fabric-interop-cc/libs/utils ################
############# END weaver/core/network/fabric-interop-cc/libs/utils ################\n
############# START weaver/core/network/fabric-interop-cc/libs/assetexchange ################
############# END weaver/core/network/fabric-interop-cc/libs/assetexchange ################\n
############# START weaver/core/network/fabric-interop-cc/interfaces/asset-mgmt ################
############# END weaver/core/network/fabric-interop-cc/interfaces/asset-mgmt ################\n
############# START weaver/core/network/fabric-interop-cc/contracts/interop ################
############# END weaver/core/network/fabric-interop-cc/contracts/interop ################\n
############# START weaver/sdks/fabric/go-sdk ################
############# END weaver/sdks/fabric/go-sdk ################\n
############# START weaver/samples/fabric/go-cli ################
############# END weaver/samples/fabric/go-cli ################\n
############# START weaver/samples/fabric/simpleasset ################
############# END weaver/samples/fabric/simpleasset ################\n
############# START weaver/samples/fabric/satpsimpleasset ################
############# END weaver/samples/fabric/satpsimpleasset ################\n
############# START weaver/samples/fabric/simpleassetandinterop ################
############# END weaver/samples/fabric/simpleassetandinterop ################\n
############# START weaver/samples/fabric/simpleassettransfer ################
############# END weaver/samples/fabric/simpleassettransfer ################\n
############# START weaver/samples/fabric/simplestatewithacl ################
############# END weaver/samples/fabric/simplestatewithacl ################\n
############# START weaver/samples/fabric/simplestate ################
############# END weaver/samples/fabric/simplestate ################\n
$ 
$ 
$ git status
On branch release-v2.1.0
Your branch is up to date with 'upstream/release-v2.1.0'.

nothing to commit, working tree clean
$ 
```

I'm going to try and investigate it more now. If you have any ideas please let me know as well!

Comment (petermetz) [reply weaver/core/network/fabric-interop-cc/contracts/interop/VERSION:1]: @sandeepnRES FYI I'm committing the following fix:
image

**Pull request merged**